docs/tutorials/estimator.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>Perhaps some words on how the different indices match up, and that they need to have the same length. Also if I didn't already know how session works, it'd be unclear to me the indices refer to the circuits/observables passed in when entering the context manager.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>